app/lib/dqt/params/trn_request.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+module Dqt
+  module Params
+    class TrnRequest
+      UNITED_KINGDOM = "United Kingdom"
+
+      GENDER_CODES = {
+        male: "Male",
+        female: "Female",
+        other: "Other",
+        gender_not_provided: "Other",
+      }.freeze
+
+      PROGRAMME_TYPE = {
+        TRAINING_ROUTE_ENUMS[:assessment_only] => "AssessmentOnlyRoute",
+        TRAINING_ROUTE_ENUMS[:early_years_assessment_only] => "EYITTAssessmentOnly",
+        TRAINING_ROUTE_ENUMS[:early_years_postgrad] => "EYITTGraduateEntry",
+        TRAINING_ROUTE_ENUMS[:early_years_salaried] => "EYITTGraduateEmploymentBased",
+        TRAINING_ROUTE_ENUMS[:early_years_undergrad] => "EYITTUndergraduate",
+        TRAINING_ROUTE_ENUMS[:hpitt_postgrad] => "TeachFirstProgramme",
+        TRAINING_ROUTE_ENUMS[:opt_in_undergrad] => "UndergraduateOptIn",
+        TRAINING_ROUTE_ENUMS[:pg_teaching_apprenticeship] => "Apprenticeship",
+        TRAINING_ROUTE_ENUMS[:provider_led_postgrad] => "Core",
+        TRAINING_ROUTE_ENUMS[:provider_led_undergrad] => "Core",
+        TRAINING_ROUTE_ENUMS[:school_direct_salaried] => "SchoolDirectTrainingProgrammeSalaried",
+        TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee] => "SchoolDirectTrainingProgrammeSelfFunded",
+      }.freeze
```

Comment:
I have mapped both provider led postgrad, and provider led undergrad to "Core". This is because in the DTTP world, we were mapping it to `6189922e-acc2-e611-80be-00155d010316`/`Provider-led` in DTTP. This route has a `dfe_code` value of "Core"

Other possible values are:
```
HEI
SchoolDirectTrainingProgramme
TeachFirstProgrammeCC
OverseasTrainedTeacherProgramme
RegisteredTeacherProgramme
GraduateTeacherProgramme
CoreFlexible
LicensedTeacherProgramme
EYITTSchoolDirectEarlyYears
FutureTeachingScholars
```

Reply:
Again, another one for @gunndabad I think! I haven't heard of this 'Core' term before.

Reply:
Checked with Rob C - mapping to 'Core' replicates current DTTP flow so this is fine.

Reply:
Checked with Rob C - mapping to 'Core' replicates current DTTP flow so this is fine.

Reply:
Checked with Rob C - mapping to 'Core' replicates current DTTP flow so this is fine.